src/main/java/com/google/ehub/servlets/ItemSubmissionServlet.java

```diff
@@ -78,19 +104,24 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
    * @param directors the directors given in the Post request parameter
    * @param writers the writers given in the Post request parameter
    * @param actors the actors given in the Post request parameter
+   * @param omdbId the unique Id used by omdb given in the Post request parameter
    * @return true if the parameters given in the Post request are valid, false otherwise
    */
   private static boolean arePostRequestParametersValid(String title, String description,
       String imageUrl, String releaseDate, String runtime, String genre, String directors,
-      String writers, String actors) {
+      String writers, String actors, String omdbId) {
     return isParameterValid(title, MAX_TITLE_CHARS) && isParameterValid(description, MAX_CHARS)
         && isParameterValid(imageUrl, MAX_CHARS) && isParameterValid(releaseDate, MAX_CHARS)
         && isParameterValid(runtime, MAX_CHARS) && isParameterValid(genre, MAX_CHARS)
         && isParameterValid(directors, MAX_CHARS) && isParameterValid(writers, MAX_CHARS)
-        && isParameterValid(actors, MAX_CHARS);
+        && isParameterValid(actors, MAX_CHARS) && isParameterValid(omdbId, MAX_CHARS);
   }
 
   private static boolean isParameterValid(String parameter, int maxLength) {
     return parameter != null && !parameter.isEmpty() && parameter.length() <= maxLength;
   }
+
+  private static boolean isItemUnique(String omdbId) {
```

Comment:
maybe doesItemExist()

Reply:
I changed the method name in last commit.